superdesk/publish/formatters/email_formatter.py

```diff
@@ -63,6 +64,7 @@ def format(self, article, subscriber, codes=None):
                 doc['message_html'] = None
             doc['message_text'] = render_template('email_article_body.txt', article=formatted_article)
             doc['message_subject'] = render_template('email_article_subject.txt', article=formatted_article)
+            doc['renditions'] = formatted_article.get('associations', {}).get('featuremedia', {}).get('renditions')
```

Comment:
Sometimes `associations.featuremedia == None`, in this case an exception will be thrown here `AttributeError: 'NoneType' object has no attribute 'get'` when attempting to get the renditions.

Reply:
OK will do!